### ISSUE TITLE
[crash] after adding resonating crystals from deep resonance as the ore sample for resonating ore

### ISSUE THREAD

**Issue by majijn** [oitsjustjose/Geolosys#131]
## Versions:

- Minecraft Forge:14.23.4.2705
- Geolosys:1.9.3
- JourneyMap (optional):-none (I do use antique atlas though)
- ImmersiveEngineering (optional):0.12.82

## What happens: I was trying to locate resonating ore to see if I was happy with how often it was spawning, I mistakenly made resonating ore the sample for resonating ore, so I changed it into resonating crystals, and teleported to the spot when I knew it was going to spawn, and then 
[crash-2018-06-12_17.12.24-server.txt](https://github.com/oitsjustjose/Geolosys/files/2094671/crash-2018-06-12_17.12.24-server.txt)

## Logs (if necessary):
[geolosys - Copy (2).cfg.txt](https://github.com/oitsjustjose/Geolosys/files/2094681/geolosys.-.Copy.2.cfg.txt)
[2018-06-12-2.log](https://github.com/oitsjustjose/Geolosys/files/2094695/2018-06-12-2.log)

**Comment by oitsjustjose** [comment]
>  what I would like to happen is the crystals would get randomized stats like the ones generated by deep resonance

Not happening. 0% chance. DeepResonance uses NBT Data to store the randomized stats, which is impractical to implement into a user config.

The crash is because of the weirdness of the block states I'm guessing. Everywhere it crashed was because the sample block could not be determined from the block state.

TL;DR: Don't use Resonating Crystals for samples. It breaks everything. Use ContentTweaker to make custom samples (Like Darkosto did in SevTech:Ages)

I *might* look into implementing DeepResonance compat myself though, that would allow this to work :)

**Comment by majijn** [comment]
that's fair! I guessed it may be something like that, I just made the report in case it may be ....well, not just the nbt weirdness
and I wasnt requesting for you to randomize the stats natively , it's just thats could happen in an ideal world LoL =)

I'll just use the ores for now (need a ton of the stuff anyway)
sorry to bother you and thank you for your time!

**Comment by oitsjustjose** [comment]
No problem! 

I'm hoping I'll soon have enough time to create modules for Geolosys that add compat for more ores, like Deep Resonance, RFTools, TechReborn, etc.
